### PR TITLE
tests + scripts

### PR DESCRIPTION
**Pull request by avniculae**

### CODE REVIEW

File: packages/tests/src/reya_network/DistributeFunds.s.sol [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import { Script } from "forge-std/Script.sol";
+import { IPassivePoolProxy } from "../interfaces/IPassivePoolProxy.sol";
+import { IERC20TokenModule } from "../interfaces/IERC20TokenModule.sol";
+import { IRUSDProxy } from "../interfaces/IRUSDProxy.sol";
+
+contract DistributeFunds is Script, Test {
+  address multisigEOA = 0x01A8e78B7ba1313A482630837c3978c6259aC1eA;
+
+  address payable pool = payable(0xB4B77d6180cc14472A9a7BDFF01cc2459368D413);
+  address usdc = 0x3B860c0b53f2e8bd5264AA7c3451d41263C933F2;
```
Comment (ioanaroceanu2000) [inline packages/tests/src/reya_network/DistributeFunds.s.sol:13]: i have another address for USDC. in my env vars, 0x1b700A40A0707Dd5f0BC31cbc57D730558E5714d
can you double check pls?

File: packages/tests/src/reya_network/DistributeFunds.s.sol [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    vm.broadcast(multisigEOA);
+    IRUSDProxy(rusd).deposit(sumDeposits);
+    vm.broadcast(multisigEOA);
+    IRUSDProxy(rusd).approve(pool, sumDeposits);
+
+    for (uint i = 0; i < pendingTxs.length; i += 1) {
+      uint256 shareSupplyUserBefore = IPassivePoolProxy(pool).getAccountBalance(poolId, pendingTxs[i].wallet_address);
+
+      vm.broadcast(multisigEOA);
+      IPassivePoolProxy(pool).addLiquidity(poolId, pendingTxs[i].wallet_address, pendingTxs[i].amount, pendingTxs[i].amount * 99 / 100 * 1e24);
```
Comment (ioanaroceanu2000) [inline packages/tests/src/reya_network/DistributeFunds.s.sol:50]: amount is with precision 6?

Comment (ioanaroceanu2000) [reply packages/tests/src/reya_network/DistributeFunds.s.sol:50]: and why 99%?

File: packages/tests/src/reya_network/DistributeFunds.s.sol (hunk 1 above)
Comment (avniculae) [reply packages/tests/src/reya_network/DistributeFunds.s.sol:13]: that's on testnet

File: packages/tests/src/reya_network/DistributeFunds.s.sol (hunk 2 above)
Comment (avniculae) [reply packages/tests/src/reya_network/DistributeFunds.s.sol:50]: amount is 6 precision, yes -- you can check the file as well, I uploaded it

Comment (avniculae) [reply packages/tests/src/reya_network/DistributeFunds.s.sol:50]: I used 99% as a sensible lower boudn limit on shares

**Pull request merged**